app/models/manageiq/providers/openstack/infra_manager.rb

```diff
@@ -64,6 +64,12 @@ def self.description
     @description ||= "OpenStack Platform Director".freeze
   end
 
+  def self.default_blacklisted_event_names
```

Comment:
@lfu weren't we capturing blacklisted events elsewhere?

We used to have a list of filtered event types.  But, I don't know how that's done now.